kanin/src/app/task.rs

```diff
@@ -100,12 +105,18 @@ async fn handle_request<H, Args, Res>(
     let properties = req.properties().cloned();
     let reply_to = properties.as_ref().and_then(|p| p.reply_to().clone());
     let correlation_id = properties.as_ref().and_then(|p| p.correlation_id().clone());
+    let app_id = properties
+        .as_ref()
+        .and_then(|p| p.app_id().as_ref())
+        .map(|app_id| app_id.as_str())
+        .unwrap_or("<unknown>");
```

Comment:
Kind of similar but I feel this should be a method in `Request`, so this can just be `req.app_id()`.

Reply:
It's definitely prettier ;)